web/middleware/cache-control.js

```diff
@@ -20,7 +20,6 @@ const STATIC_ASSET_PATHS = [
   '/public/img/thumbnail-broken.png',
   '/public/img/thumbnail-missing.png',
   '/public/img/total-background.png',
-  '/public/img/cookie.svg',
```

Comment:
Still need this -- I believe we can't place it in the CDN because of domain issues where the image is used.

Reply:
Ah, this might explain things. Webpack is doing more static analysis, so this might need to be enumerated to be ignored.

Reply:
Actually, poking through further, do you know where I can find this for reference? After casually browsing around and watching the network tab, I can't seem to find any reference to the file.

Reply:
`--- close ---`
Same as https://github.com/OdyseeTeam/odysee-frontend/pull/1902#discussion_r931057729